cmd/virt-freezer/main.go

```diff
@@ -102,7 +102,7 @@ func main() {
 	log.Log.Infof("Guest agent version is %s", info.GAVersion)
 
 	if !isVmRunning(client) {
-		log.Log.Reason(err).Error("VM is not running")
+		log.Log.Reason(err).Error("Paused VM, unable to freeze/unfreeze")
```

Comment:
Why are we logging `err` from L91 here?

Reply:
Good point! don't know how I missed that.